05/Weather.js

```diff
@@ -10,14 +11,11 @@ class Weather extends React.Component{
     }
     componentDidMount(){
         const {signal} = this.abortController;
-        return fetch(`https://api.weatherbit.io/v2.0/current?key=93558fd78ea34b9182d933c681a32e9b&lat=${this.props.lat}&lon=${this.props.lon}&lang=pl`, {signal})
-        .then((response) => response.json())
-        .then((data) => {
-            this.setState({data: data});
-        })
-        .catch((error) => {
-            console.error(error);
-        })
+        const url = `https://api.weatherbit.io/v2.0/current?key=93558fd78ea34b9182d933c681a32e9b&lat=${this.props.lat}&lon=${this.props.lon}&lang=pl`
```

Comment:
`url` też bym przerzucił do `WeatherProvider.js` - jeśli coś się zmieni w API (też url) to chciałbym zmieniać to w tym jednym pliku ;)

Reply:
Tylko jak wtedy przekazywać do url props lat i lon?